src/board/mod.rs

```diff
@@ -563,6 +563,41 @@ impl Board {
         self.board[c.to_index(self.size)].chain_id = -1;
         self.board[c.to_index(self.size)].color = Empty;
     }
+    
+    pub fn liberty_count(&self, c: Coord) -> usize {
```

Comment:
I added tests and now I do non-overlapping liberties which is slower, but more correct